ci/jenkins/jobs/projects-lab.yaml

```diff
@@ -20,6 +20,19 @@
           ignore_post_commit_hooks: false
           publishers: []
           wrappers: []
+      - '{name}-{test_name}-for-period':
+          test_name: jenkins-backup
+          node: 'backup-node'
+          description: 'This is for backing up all jenkins files.'
+          builders:
+            - builder-jenkins-backup:
+                rdiff_username: '{jenkins_username}'
```

Comment:
Jenkins-job-builder variable will be render into plan text. I suggest using credential binding instead of Jenkins-job-builder variable. There are some examples in this file such as credentials for govc .

Reply:
We may need to create multiple vm for both lab and cloud jenkins, which means they will have different IPs. Do you think we should add one more job in cloud yaml?

Reply:
Updated, thanks.